results.csv

```diff
@@ -1,18 +1,33 @@
-VehicleID,Delayed in hours
```

Comment:
Löscht die `results.csv` mal in diesem PR und fügt sie dafür in die `.gitignore` ein.